sentinel-extension/sentinel-datasource-redis/src/main/java/com/alibaba/csp/sentinel/datasource/redis/RedisDataSource.java

```diff
@@ -0,0 +1,82 @@
+package com.alibaba.csp.sentinel.datasource.redis;
+
+import com.alibaba.csp.sentinel.datasource.AbstractDataSource;
+import com.alibaba.csp.sentinel.datasource.ConfigParser;
+import com.alibaba.csp.sentinel.log.RecordLog;
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.api.sync.RedisCommands;
+import io.lettuce.core.pubsub.RedisPubSubAdapter;
+import io.lettuce.core.pubsub.StatefulRedisPubSubConnection;
+import io.lettuce.core.pubsub.api.sync.RedisPubSubCommands;
+
+
+/**
+ * Redis data resource. when data publish in redis,this will update rule config in time
+ *
+ * @author moon tiger
+ */
+
```

Comment:
You can reformat your code with Alibaba P3C Plugin and obey the rules of Alibaba Coding Guidelines. (e.g. remove redundant lines)